index.js

```diff
@@ -221,11 +242,17 @@ class MarkdownToolbarElement extends HTMLElement {
   }
 
   connectedCallback() {
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'toolbar')
+    }
+    this.addEventListener('keydown', focusKeydown)
     const fn = shortcut.bind(null, this)
     if (this.field) {
       this.field.addEventListener('keydown', fn)
       shortcutListeners.set(this, fn)
     }
+    this.setAttribute('tabindex', '0')
+    this.addEventListener('focus', onToolbarFocus)
```

Comment:
I see tabindex is removed, but just for good measure-

```suggestion
    this.addEventListener('focus', onToolbarFocus, {once: true})
```